Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -93,7 +93,14 @@ gdjs.TweenRuntimeBehavior.prototype._pauseTween = function(identifier) {
 };
 
 gdjs.TweenRuntimeBehavior.prototype._resumeTween = function(identifier) {
-  return this._tweens[identifier].instance.resume();
+  return this._tweens[identifier].instance
```

Comment:
Nice 👍

Reply:
Unfortunately this didn't solve the issue :cry: